package.json

```diff
@@ -68,7 +67,7 @@
 		"arrowParens": "avoid"
 	},
 	"lint-staged": {
-		"packages/**/{src,test}/**/*.js": [
+		"packages/**/*.{js,ts}": [
```

Comment:
I did have to run `npm rebuild` / `npm rebuild --update-binary` in order to make Husky play nice. Lint staged kept strangely erroring out otherwise.